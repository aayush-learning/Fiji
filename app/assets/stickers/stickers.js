const stickers = {
  "unknown.svg": require('./caterpillar.svg'),
  "caterpillar.svg": require('./caterpillar.svg'),
  "caterpillar_walk.svg": require('./caterpillar_walk.svg'),
  "caterpillar_dance.svg": require('./caterpillar_dance.svg'),
  "cheshire-cat.svg": require('./cheshire-cat.svg'),
  "cheshire-cat_grin.svg": require('./cheshire-cat_grin.svg'),
  "cheshire-cat_clap.svg": require('./cheshire-cat_clap.svg')
}

export default stickers;