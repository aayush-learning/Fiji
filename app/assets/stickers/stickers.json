{
  "caterpillar.svg": [
    "caterpillar_walk.svg",
    "caterpillar_dance.svg"
  ],
  "cheshire-cat.svg": [
    "cheshire-cat_grin.svg",
    "cheshire-cat_clap.svg"
  ]
}