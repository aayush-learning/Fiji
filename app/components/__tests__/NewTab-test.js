import 'react-native';
import React from 'react';
import FacebookTabBar from '../NewTab';
import stickers from '../../assets/stickers/stickers.json'

import renderer from 'react-test-renderer';

//npm test -- --updateSnapshot
//this above command is used to update the snapshot in windows

//npm test -- --coverage
//this above command is used to see the total coverage of the tested file in windows


//jest --updateSnapshot
//this above command is used to update the snapshot in IOS

//jest --coverage
//this above command is used to see the total coverage of the tested file in IOS

//npm test -- --coverage User-test
//this above command is used to test indivisual component

it('this is user 0', () => {
  const tree = renderer.create(
  <FacebookTabBar tabData={stickers} />
).toJSON();
  expect(tree).toMatchSnapshot();
});

