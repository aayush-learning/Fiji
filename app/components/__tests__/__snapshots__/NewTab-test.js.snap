// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`this is user 0 1`] = `
<View
  style={
    Array [
      Object {
        "borderBottomColor": "red",
        "borderLeftWidth": 0,
        "borderRightWidth": 0,
        "borderTopWidth": 0,
        "borderWidth": 1,
        "flexDirection": "row",
        "height": 45,
        "paddingTop": 5,
      },
      undefined,
    ]
  }
>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    collapsable={undefined}
    hasTVPreferredFocus={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
        "opacity": 1,
        "paddingBottom": 10,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <View
      style={undefined}
    >
      <RNSVGSvgView
        align="xMidYMid"
        enableBackground="new 0 0 512 512"
        meetOrSlice={0}
        minX={0}
        minY={0}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            undefined,
            false,
            Object {
              "flex": 0,
              "height": 30,
              "width": 30,
            },
          ]
        }
        vbHeight={512}
        vbWidth={512}
        x="0"
        y="0"
      >
        <RNSVGPath
          d="M364.042,329.185c13.205-8.103,29.165-12.761,46.068-12.761c49.723,0,89.961,40.312,89.961,90.035
	s-40.199,90.035-89.922,90.035c-16.903,0-32.9-4.658-46.105-12.761V329.185H364.042z"
          fill={
            Array [
              0,
              0.2823529411764706,
              0.596078431372549,
              0.8549019607843137,
              1,
            ]
          }
          fillOpacity={1}
          fillRule={1}
          matrix={
            Array [
              1,
              0,
              0,
              1,
              0,
              0,
            ]
          }
          opacity={1}
          originX={0}
          originY={0}
          propList={
            Array [
              "fill",
            ]
          }
          rotation={0}
          scaleX={1}
          scaleY={1}
          skewX={0}
          skewY={0}
          stroke={null}
          strokeDasharray={null}
          strokeDashoffset={null}
          strokeLinecap={0}
          strokeLinejoin={0}
          strokeMiterlimit={4}
          strokeOpacity={1}
          strokeWidth="1"
          x={0}
          y={0}
        />
        <RNSVGPath
          d="M272.807,328.478c13.205-7.659,28.469-12.053,44.868-12.053c16.903,0,32.638,4.658,46.144,12.761
	c26.254,15.726,43.78,44.453,43.78,77.274s-17.582,61.548-43.836,77.274c-13.505,8.103-29.175,12.761-46.078,12.761
	c-16.398,0-31.672-4.394-44.877-12.053V328.478z"
          fill={
            Array [
              0,
              0.9725490196078431,
              0.6588235294117647,
              0.0196078431372549,
              1,
            ]
          }
          fillOpacity={1}
          fillRule={1}
          matrix={
            Array [
              1,
              0,
              0,
              1,
              0,
              0,
            ]
          }
          opacity={1}
          originX={0}
          originY={0}
          propList={
            Array [
              "fill",
            ]
          }
          rotation={0}
          scaleX={1}
          scaleY={1}
          skewX={0}
          skewY={0}
          stroke={null}
          strokeDasharray={null}
          strokeDashoffset={null}
          strokeLinecap={0}
          strokeLinejoin={0}
          strokeMiterlimit={4}
          strokeOpacity={1}
          strokeWidth="1"
          x={0}
          y={0}
        />
        <RNSVGPath
          d="M223.851,316.509c1.212-0.06,2.425-0.084,3.637-0.084c16.398,0,31.776,4.394,45.018,12.053
	c26.915,15.558,45.018,44.657,45.018,77.982s-18.103,62.424-45.018,77.982c-13.241,7.659-28.619,12.053-45.018,12.053
	c-17.143,0-33.181-4.802-46.818-13.121L223.851,316.509z"
          fill={
            Array [
              0,
              0.24313725490196078,
              0.6862745098039216,
              0.39215686274509803,
              1,
            ]
          }
          fillOpacity={1}
          fillRule={1}
          matrix={
            Array [
              1,
              0,
              0,
              1,
              0,
              0,
            ]
          }
          opacity={1}
          originX={0}
          originY={0}
          propList={
            Array [
              "fill",
            ]
          }
          rotation={0}
          scaleX={1}
          scaleY={1}
          skewX={0}
          skewY={0}
          stroke={null}
          strokeDasharray={null}
          strokeDashoffset={null}
          strokeLinecap={0}
          strokeLinejoin={0}
          strokeMiterlimit={4}
          strokeOpacity={1}
          strokeWidth="1"
          x={0}
          y={0}
        />
        <RNSVGPath
          d="M212.507,362.842c7.251,13.205,11.38,27.811,11.38,43.717c0,32.581-17.299,61.067-43.217,76.864
	c-13.637,8.319-29.676,13.096-46.818,13.096c-49.723,0-90.035-40.224-90.035-89.948c0-15.906,4.13-30.524,11.38-43.729H212.507z"
          fill={
            Array [
              0,
              0.8235294117647058,
              0.3215686274509804,
              0.33725490196078434,
              1,
            ]
          }
          fillOpacity={1}
          fillRule={1}
          matrix={
            Array [
              1,
              0,
              0,
              1,
              0,
              0,
            ]
          }
          opacity={1}
          originX={0}
          originY={0}
          propList={
            Array [
              "fill",
            ]
          }
          rotation={0}
          scaleX={1}
          scaleY={1}
          skewX={0}
          skewY={0}
          stroke={null}
          strokeDasharray={null}
          strokeDashoffset={null}
          strokeLinecap={0}
          strokeLinejoin={0}
          strokeMiterlimit={4}
          strokeOpacity={1}
          strokeWidth="1"
          x={0}
          y={0}
        />
        <RNSVGPath
          d="M212.171,274.008c7.095,13.205,11.296,27.007,11.681,42.301c0.024,0.768,0.036,1.636,0.036,2.417
	c0,15.906-4.13,30.902-11.38,43.868c-14.826,26.59-42.737,44.899-75.029,46.171c-1.2,0.048-2.413,0.084-3.625,0.084
	c-33.817,0-63.277-18.637-78.655-46.212c-7.251-12.965-11.38-28.108-11.38-44.014c0-16.158,4.262-31.409,11.717-44.614H212.171z"
          fill={
            Array [
              0,
              0.2823529411764706,
              0.596078431372549,
              0.8549019607843137,
              1,
            ]
          }
          fillOpacity={1}
          fillRule={1}
          matrix={
            Array [
              1,
              0,
              0,
              1,
              0,
              0,
            ]
          }
          opacity={1}
          originX={0}
          originY={0}
          propList={
            Array [
              "fill",
            ]
          }
          rotation={0}
          scaleX={1}
          scaleY={1}
          skewX={0}
          skewY={0}
          stroke={null}
          strokeDasharray={null}
          strokeDashoffset={null}
          strokeLinecap={0}
          strokeLinejoin={0}
          strokeMiterlimit={4}
          strokeOpacity={1}
          strokeWidth="1"
          x={0}
          y={0}
        />
        <RNSVGPath
          d="M223.887,229.99c0,16.158-4.262,31.308-11.717,44.417c-15.474,27.239-44.753,45.618-78.319,45.618
	s-62.845-18.379-78.319-45.618c-7.455-13.109-11.717-28.259-11.717-44.417c0-49.723,40.312-90.035,90.035-90.035
	S223.887,180.268,223.887,229.99z"
          fill={
            Array [
              0,
              0.9725490196078431,
              0.6588235294117647,
              0.0196078431372549,
              1,
            ]
          }
          fillOpacity={1}
          fillRule={1}
          matrix={
            Array [
              1,
              0,
              0,
              1,
              0,
              0,
            ]
          }
          opacity={1}
          originX={0}
          originY={0}
          propList={
            Array [
              "fill",
            ]
          }
          rotation={0}
          scaleX={1}
          scaleY={1}
          skewX={0}
          skewY={0}
          stroke={null}
          strokeDasharray={null}
          strokeDashoffset={null}
          strokeLinecap={0}
          strokeLinejoin={0}
          strokeMiterlimit={4}
          strokeOpacity={1}
          strokeWidth="1"
          x={0}
          y={0}
        />
        <RNSVGGroup
          fill={
            Array [
              0,
              0,
              0,
              0,
              1,
            ]
          }
          fillOpacity={1}
          fillRule={1}
          font={Object {}}
          matrix={
            Array [
              1,
              0,
              0,
              1,
              0,
              0,
            ]
          }
          opacity={1}
          originX={0}
          originY={0}
          propList={Array []}
          rotation={0}
          scaleX={1}
          scaleY={1}
          skewX={0}
          skewY={0}
          stroke={null}
          strokeDasharray={null}
          strokeDashoffset={null}
          strokeLinecap={0}
          strokeLinejoin={0}
          strokeMiterlimit={4}
          strokeOpacity={1}
          strokeWidth="1"
          x={0}
          y={0}
        >
          <RNSVGPath
            d="M60.023,111.545c-26.478,0-48.019-21.541-48.019-48.019s21.541-48.019,48.019-48.019
		s48.019,21.541,48.019,48.019S86.501,111.545,60.023,111.545z"
            fill={
              Array [
                0,
                0.9254901960784314,
                0.9411764705882353,
                0.9450980392156862,
                1,
              ]
            }
            fillOpacity={1}
            fillRule={1}
            matrix={
              Array [
                1,
                0,
                0,
                1,
                0,
                0,
              ]
            }
            opacity={1}
            originX={0}
            originY={0}
            propList={
              Array [
                "fill",
              ]
            }
            rotation={0}
            scaleX={1}
            scaleY={1}
            skewX={0}
            skewY={0}
            stroke={null}
            strokeDasharray={null}
            strokeDashoffset={null}
            strokeLinecap={0}
            strokeLinejoin={0}
            strokeMiterlimit={4}
            strokeOpacity={1}
            strokeWidth="1"
            x={0}
            y={0}
          />
          <RNSVGPath
            d="M207.681,111.545c-26.478,0-48.019-21.541-48.019-48.019s21.541-48.019,48.019-48.019
		S255.7,37.048,255.7,63.526S234.159,111.545,207.681,111.545z"
            fill={
              Array [
                0,
                0.9254901960784314,
                0.9411764705882353,
                0.9450980392156862,
                1,
              ]
            }
            fillOpacity={1}
            fillRule={1}
            matrix={
              Array [
                1,
                0,
                0,
                1,
                0,
                0,
              ]
            }
            opacity={1}
            originX={0}
            originY={0}
            propList={
              Array [
                "fill",
              ]
            }
            rotation={0}
            scaleX={1}
            scaleY={1}
            skewX={0}
            skewY={0}
            stroke={null}
            strokeDasharray={null}
            strokeDashoffset={null}
            strokeLinecap={0}
            strokeLinejoin={0}
            strokeMiterlimit={4}
            strokeOpacity={1}
            strokeWidth="1"
            x={0}
            y={0}
          />
        </RNSVGGroup>
        <RNSVGGroup
          fill={
            Array [
              0,
              0,
              0,
              0,
              1,
            ]
          }
          fillOpacity={1}
          fillRule={1}
          font={Object {}}
          matrix={
            Array [
              1,
              0,
              0,
              1,
              0,
              0,
            ]
          }
          opacity={1}
          originX={0}
          originY={0}
          propList={Array []}
          rotation={0}
          scaleX={1}
          scaleY={1}
          skewX={0}
          skewY={0}
          stroke={null}
          strokeDasharray={null}
          strokeDashoffset={null}
          strokeLinecap={0}
          strokeLinejoin={0}
          strokeMiterlimit={4}
          strokeOpacity={1}
          strokeWidth="1"
          x={0}
          y={0}
        >
          <RNSVGPath
            d="M409.96,304.419c-16.128,0-31.966,3.807-46.218,11.056c-14.252-7.25-30.09-11.056-46.218-11.056
		c-15.642,0-31.071,3.6-45.018,10.457c-11.741-5.772-24.529-9.228-37.613-10.179c-1.469-10.498-4.567-20.745-9.175-30.283
		c6.671-13.787,10.174-29.012,10.174-44.425c0-43.62-27.521-80.919-66.104-95.488l15.36-15.36
		c6.963,2.832,14.566,4.406,22.534,4.406c33.097,0,60.023-26.927,60.023-60.023S240.779,3.501,207.682,3.501
		s-60.023,26.927-60.023,60.023c0,16.328,6.561,31.145,17.175,41.978l-22.805,22.805c-2.7-0.215-5.421-0.357-8.175-0.357
		c-3.787,0-7.523,0.224-11.205,0.628l-21.446-21.445c11.591-10.949,18.845-26.443,18.845-43.608
		c0-33.097-26.927-60.023-60.023-60.023S0,30.429,0,63.526s26.927,60.023,60.023,60.023c7.066,0,13.846-1.236,20.149-3.49
		l15.379,15.379c-37.337,15.179-63.739,51.83-63.739,94.551c0,15.412,3.502,30.635,10.17,44.417
		c-6.669,13.785-10.17,29.008-10.17,44.417c0,15.192,3.403,30.212,9.881,43.817c-6.479,13.612-9.881,28.629-9.881,43.817
		c0,56.265,45.775,102.04,102.04,102.04c16.359,0,32.404-3.912,46.818-11.364c14.416,7.451,30.46,11.364,46.818,11.364
		c15.643,0,31.072-3.6,45.018-10.457c13.947,6.857,29.375,10.457,45.018,10.457c16.131,0,31.971-3.808,46.219-11.056
		c14.252,7.25,30.089,11.056,46.217,11.056c56.265,0,102.04-45.775,102.04-102.04S466.225,304.419,409.96,304.419z M207.681,27.512
		c19.858,0,36.014,16.156,36.014,36.014S227.539,99.54,207.681,99.54s-36.014-16.156-36.014-36.014S187.823,27.512,207.681,27.512z
		 M24.009,63.526c0-19.858,16.156-36.014,36.014-36.014s36.014,16.156,36.014,36.014c0,13.503-7.474,25.287-18.497,31.451
		c-0.261,0.128-0.515,0.269-0.767,0.417c-5.007,2.642-10.705,4.146-16.75,4.146C40.165,99.54,24.009,83.383,24.009,63.526z
		 M65.675,356.782c-6.447-11.525-9.853-24.649-9.853-37.956c0-6.817,0.898-13.586,2.637-20.119
		c19.123,20.937,46.435,33.324,75.393,33.324c28.966,0,56.285-12.394,75.409-33.341c1.562,5.872,2.435,11.947,2.591,18.203
		c0.022,0.64,0.03,1.281,0.03,1.932c0,13.307-3.407,26.433-9.862,37.971c-13.362,23.967-37.667,38.919-65.037,39.997
		c-1.037,0.042-2.084,0.062-3.132,0.062C105.584,396.855,79.463,381.505,65.675,356.782z M133.852,151.96
		c43.026,0,78.03,35.004,78.03,78.03c0,13.503-3.509,26.81-10.149,38.488c-13.858,24.39-39.869,39.542-67.882,39.542
		s-54.023-15.152-67.884-39.547c-6.637-11.673-10.146-24.981-10.146-38.483C55.822,186.965,90.826,151.96,133.852,151.96z
		 M55.822,406.459c0-6.489,0.816-12.931,2.394-19.169c19.117,21.101,46.501,33.575,75.637,33.575c1.393,0,2.777-0.028,4.097-0.082
		c27.468-1.082,53.319-13.493,71.534-33.521c1.582,6.246,2.4,12.699,2.4,19.197c0,27.444-14.003,52.366-37.464,66.666
		c-12.186,7.435-26.215,11.365-40.566,11.365C90.826,484.489,55.822,449.485,55.822,406.459z M203.783,480.814
		c20.384-19.06,32.109-45.647,32.109-74.355c0-15.188-3.402-30.205-9.881-43.817c5.05-10.611,8.228-22.075,9.385-33.809
		c10.949,1.113,21.572,4.525,31.102,10.036c24.069,13.913,39.021,39.812,39.021,67.59c0,27.778-14.952,53.677-39.024,67.591
		c-11.808,6.829-25.296,10.439-39.007,10.439C219.393,484.489,211.4,483.239,203.783,480.814z M296.486,481.598
		c20.757-19.119,33.043-46.353,33.043-75.137c0-28.786-12.287-56.02-33.043-75.137c6.815-1.904,13.895-2.893,21.038-2.893
		c14.146,0,27.993,3.821,40.049,11.055c0.016,0.01,0.031,0.019,0.047,0.029c0.006,0.004,0.012,0.008,0.019,0.012
		c23.743,14.253,37.916,39.269,37.916,66.935c0,27.66-14.167,52.671-37.907,66.93c-0.016,0.01-0.031,0.02-0.048,0.03
		c-0.011,0.006-0.022,0.014-0.034,0.02c-12.049,7.229-25.895,11.05-40.042,11.05C310.381,484.489,303.301,483.501,296.486,481.598z
		 M409.96,484.489c-7.804,0-15.516-1.169-22.887-3.426c20.402-19.113,32.491-46.183,32.491-74.604
		c0-28.423-12.09-55.494-32.489-74.604c7.371-2.257,15.081-3.426,22.886-3.426c43.026,0,78.03,35.004,78.03,78.03
		S452.986,484.489,409.96,484.489z"
            fill={
              Array [
                0,
                0.13725490196078433,
                0.12156862745098039,
                0.12549019607843137,
                1,
              ]
            }
            fillOpacity={1}
            fillRule={1}
            matrix={
              Array [
                1,
                0,
                0,
                1,
                0,
                0,
              ]
            }
            opacity={1}
            originX={0}
            originY={0}
            propList={
              Array [
                "fill",
              ]
            }
            rotation={0}
            scaleX={1}
            scaleY={1}
            skewX={0}
            skewY={0}
            stroke={null}
            strokeDasharray={null}
            strokeDashoffset={null}
            strokeLinecap={0}
            strokeLinejoin={0}
            strokeMiterlimit={4}
            strokeOpacity={1}
            strokeWidth="1"
            x={0}
            y={0}
          />
          <RNSVGCircle
            cx="60.023"
            cy="63.526"
            fill={
              Array [
                0,
                0.13725490196078433,
                0.12156862745098039,
                0.12549019607843137,
                1,
              ]
            }
            fillOpacity={1}
            fillRule={1}
            matrix={
              Array [
                1,
                0,
                0,
                1,
                0,
                0,
              ]
            }
            opacity={1}
            originX={0}
            originY={0}
            propList={
              Array [
                "fill",
              ]
            }
            r="12.005"
            rotation={0}
            scaleX={1}
            scaleY={1}
            skewX={0}
            skewY={0}
            stroke={null}
            strokeDasharray={null}
            strokeDashoffset={null}
            strokeLinecap={0}
            strokeLinejoin={0}
            strokeMiterlimit={4}
            strokeOpacity={1}
            strokeWidth="1"
            x={0}
            y={0}
          />
          <RNSVGCircle
            cx="207.681"
            cy="63.526"
            fill={
              Array [
                0,
                0.13725490196078433,
                0.12156862745098039,
                0.12549019607843137,
                1,
              ]
            }
            fillOpacity={1}
            fillRule={1}
            matrix={
              Array [
                1,
                0,
                0,
                1,
                0,
                0,
              ]
            }
            opacity={1}
            originX={0}
            originY={0}
            propList={
              Array [
                "fill",
              ]
            }
            r="12.005"
            rotation={0}
            scaleX={1}
            scaleY={1}
            skewX={0}
            skewY={0}
            stroke={null}
            strokeDasharray={null}
            strokeDashoffset={null}
            strokeLinecap={0}
            strokeLinejoin={0}
            strokeMiterlimit={4}
            strokeOpacity={1}
            strokeWidth="1"
            x={0}
            y={0}
          />
          <RNSVGPath
            d="M133.852,284.412c24.824,0,46.978-17.7,52.678-42.086c1.509-6.456-2.501-12.913-8.957-14.422
		c-6.463-1.516-12.913,2.502-14.422,8.957c-3.137,13.421-15.732,23.541-29.299,23.541s-26.162-10.121-29.299-23.541
		c-1.51-6.457-7.968-10.464-14.422-8.957c-6.456,1.509-10.466,7.966-8.957,14.422C86.874,266.712,109.028,284.412,133.852,284.412z"
            fill={
              Array [
                0,
                0.13725490196078433,
                0.12156862745098039,
                0.12549019607843137,
                1,
              ]
            }
            fillOpacity={1}
            fillRule={1}
            matrix={
              Array [
                1,
                0,
                0,
                1,
                0,
                0,
              ]
            }
            opacity={1}
            originX={0}
            originY={0}
            propList={
              Array [
                "fill",
              ]
            }
            rotation={0}
            scaleX={1}
            scaleY={1}
            skewX={0}
            skewY={0}
            stroke={null}
            strokeDasharray={null}
            strokeDashoffset={null}
            strokeLinecap={0}
            strokeLinejoin={0}
            strokeMiterlimit={4}
            strokeOpacity={1}
            strokeWidth="1"
            x={0}
            y={0}
          />
        </RNSVGGroup>
        <RNSVGGroup
          fill={
            Array [
              0,
              0,
              0,
              0,
              1,
            ]
          }
          fillOpacity={1}
          fillRule={1}
          font={Object {}}
          matrix={
            Array [
              1,
              0,
              0,
              1,
              0,
              0,
            ]
          }
          opacity={1}
          originX={0}
          originY={0}
          propList={Array []}
          rotation={0}
          scaleX={1}
          scaleY={1}
          skewX={0}
          skewY={0}
          stroke={null}
          strokeDasharray={null}
          strokeDashoffset={null}
          strokeLinecap={0}
          strokeLinejoin={0}
          strokeMiterlimit={4}
          strokeOpacity={1}
          strokeWidth="1"
          x={0}
          y={0}
        />
        <RNSVGGroup
          fill={
            Array [
              0,
              0,
              0,
              0,
              1,
            ]
          }
          fillOpacity={1}
          fillRule={1}
          font={Object {}}
          matrix={
            Array [
              1,
              0,
              0,
              1,
              0,
              0,
            ]
          }
          opacity={1}
          originX={0}
          originY={0}
          propList={Array []}
          rotation={0}
          scaleX={1}
          scaleY={1}
          skewX={0}
          skewY={0}
          stroke={null}
          strokeDasharray={null}
          strokeDashoffset={null}
          strokeLinecap={0}
          strokeLinejoin={0}
          strokeMiterlimit={4}
          strokeOpacity={1}
          strokeWidth="1"
          x={0}
          y={0}
        />
        <RNSVGGroup
          fill={
            Array [
              0,
              0,
              0,
              0,
              1,
            ]
          }
          fillOpacity={1}
          fillRule={1}
          font={Object {}}
          matrix={
            Array [
              1,
              0,
              0,
              1,
              0,
              0,
            ]
          }
          opacity={1}
          originX={0}
          originY={0}
          propList={Array []}
          rotation={0}
          scaleX={1}
          scaleY={1}
          skewX={0}
          skewY={0}
          stroke={null}
          strokeDasharray={null}
          strokeDashoffset={null}
          strokeLinecap={0}
          strokeLinejoin={0}
          strokeMiterlimit={4}
          strokeOpacity={1}
          strokeWidth="1"
          x={0}
          y={0}
        />
        <RNSVGGroup
          fill={
            Array [
              0,
              0,
              0,
              0,
              1,
            ]
          }
          fillOpacity={1}
          fillRule={1}
          font={Object {}}
          matrix={
            Array [
              1,
              0,
              0,
              1,
              0,
              0,
            ]
          }
          opacity={1}
          originX={0}
          originY={0}
          propList={Array []}
          rotation={0}
          scaleX={1}
          scaleY={1}
          skewX={0}
          skewY={0}
          stroke={null}
          strokeDasharray={null}
          strokeDashoffset={null}
          strokeLinecap={0}
          strokeLinejoin={0}
          strokeMiterlimit={4}
          strokeOpacity={1}
          strokeWidth="1"
          x={0}
          y={0}
        />
        <RNSVGGroup
          fill={
            Array [
              0,
              0,
              0,
              0,
              1,
            ]
          }
          fillOpacity={1}
          fillRule={1}
          font={Object {}}
          matrix={
            Array [
              1,
              0,
              0,
              1,
              0,
              0,
            ]
          }
          opacity={1}
          originX={0}
          originY={0}
          propList={Array []}
          rotation={0}
          scaleX={1}
          scaleY={1}
          skewX={0}
          skewY={0}
          stroke={null}
          strokeDasharray={null}
          strokeDashoffset={null}
          strokeLinecap={0}
          strokeLinejoin={0}
          strokeMiterlimit={4}
          strokeOpacity={1}
          strokeWidth="1"
          x={0}
          y={0}
        />
        <RNSVGGroup
          fill={
            Array [
              0,
              0,
              0,
              0,
              1,
            ]
          }
          fillOpacity={1}
          fillRule={1}
          font={Object {}}
          matrix={
            Array [
              1,
              0,
              0,
              1,
              0,
              0,
            ]
          }
          opacity={1}
          originX={0}
          originY={0}
          propList={Array []}
          rotation={0}
          scaleX={1}
          scaleY={1}
          skewX={0}
          skewY={0}
          stroke={null}
          strokeDasharray={null}
          strokeDashoffset={null}
          strokeLinecap={0}
          strokeLinejoin={0}
          strokeMiterlimit={4}
          strokeOpacity={1}
          strokeWidth="1"
          x={0}
          y={0}
        />
        <RNSVGGroup
          fill={
            Array [
              0,
              0,
              0,
              0,
              1,
            ]
          }
          fillOpacity={1}
          fillRule={1}
          font={Object {}}
          matrix={
            Array [
              1,
              0,
              0,
              1,
              0,
              0,
            ]
          }
          opacity={1}
          originX={0}
          originY={0}
          propList={Array []}
          rotation={0}
          scaleX={1}
          scaleY={1}
          skewX={0}
          skewY={0}
          stroke={null}
          strokeDasharray={null}
          strokeDashoffset={null}
          strokeLinecap={0}
          strokeLinejoin={0}
          strokeMiterlimit={4}
          strokeOpacity={1}
          strokeWidth="1"
          x={0}
          y={0}
        />
        <RNSVGGroup
          fill={
            Array [
              0,
              0,
              0,
              0,
              1,
            ]
          }
          fillOpacity={1}
          fillRule={1}
          font={Object {}}
          matrix={
            Array [
              1,
              0,
              0,
              1,
              0,
              0,
            ]
          }
          opacity={1}
          originX={0}
          originY={0}
          propList={Array []}
          rotation={0}
          scaleX={1}
          scaleY={1}
          skewX={0}
          skewY={0}
          stroke={null}
          strokeDasharray={null}
          strokeDashoffset={null}
          strokeLinecap={0}
          strokeLinejoin={0}
          strokeMiterlimit={4}
          strokeOpacity={1}
          strokeWidth="1"
          x={0}
          y={0}
        />
        <RNSVGGroup
          fill={
            Array [
              0,
              0,
              0,
              0,
              1,
            ]
          }
          fillOpacity={1}
          fillRule={1}
          font={Object {}}
          matrix={
            Array [
              1,
              0,
              0,
              1,
              0,
              0,
            ]
          }
          opacity={1}
          originX={0}
          originY={0}
          propList={Array []}
          rotation={0}
          scaleX={1}
          scaleY={1}
          skewX={0}
          skewY={0}
          stroke={null}
          strokeDasharray={null}
          strokeDashoffset={null}
          strokeLinecap={0}
          strokeLinejoin={0}
          strokeMiterlimit={4}
          strokeOpacity={1}
          strokeWidth="1"
          x={0}
          y={0}
        />
        <RNSVGGroup
          fill={
            Array [
              0,
              0,
              0,
              0,
              1,
            ]
          }
          fillOpacity={1}
          fillRule={1}
          font={Object {}}
          matrix={
            Array [
              1,
              0,
              0,
              1,
              0,
              0,
            ]
          }
          opacity={1}
          originX={0}
          originY={0}
          propList={Array []}
          rotation={0}
          scaleX={1}
          scaleY={1}
          skewX={0}
          skewY={0}
          stroke={null}
          strokeDasharray={null}
          strokeDashoffset={null}
          strokeLinecap={0}
          strokeLinejoin={0}
          strokeMiterlimit={4}
          strokeOpacity={1}
          strokeWidth="1"
          x={0}
          y={0}
        />
        <RNSVGGroup
          fill={
            Array [
              0,
              0,
              0,
              0,
              1,
            ]
          }
          fillOpacity={1}
          fillRule={1}
          font={Object {}}
          matrix={
            Array [
              1,
              0,
              0,
              1,
              0,
              0,
            ]
          }
          opacity={1}
          originX={0}
          originY={0}
          propList={Array []}
          rotation={0}
          scaleX={1}
          scaleY={1}
          skewX={0}
          skewY={0}
          stroke={null}
          strokeDasharray={null}
          strokeDashoffset={null}
          strokeLinecap={0}
          strokeLinejoin={0}
          strokeMiterlimit={4}
          strokeOpacity={1}
          strokeWidth="1"
          x={0}
          y={0}
        />
        <RNSVGGroup
          fill={
            Array [
              0,
              0,
              0,
              0,
              1,
            ]
          }
          fillOpacity={1}
          fillRule={1}
          font={Object {}}
          matrix={
            Array [
              1,
              0,
              0,
              1,
              0,
              0,
            ]
          }
          opacity={1}
          originX={0}
          originY={0}
          propList={Array []}
          rotation={0}
          scaleX={1}
          scaleY={1}
          skewX={0}
          skewY={0}
          stroke={null}
          strokeDasharray={null}
          strokeDashoffset={null}
          strokeLinecap={0}
          strokeLinejoin={0}
          strokeMiterlimit={4}
          strokeOpacity={1}
          strokeWidth="1"
          x={0}
          y={0}
        />
        <RNSVGGroup
          fill={
            Array [
              0,
              0,
              0,
              0,
              1,
            ]
          }
          fillOpacity={1}
          fillRule={1}
          font={Object {}}
          matrix={
            Array [
              1,
              0,
              0,
              1,
              0,
              0,
            ]
          }
          opacity={1}
          originX={0}
          originY={0}
          propList={Array []}
          rotation={0}
          scaleX={1}
          scaleY={1}
          skewX={0}
          skewY={0}
          stroke={null}
          strokeDasharray={null}
          strokeDashoffset={null}
          strokeLinecap={0}
          strokeLinejoin={0}
          strokeMiterlimit={4}
          strokeOpacity={1}
          strokeWidth="1"
          x={0}
          y={0}
        />
        <RNSVGGroup
          fill={
            Array [
              0,
              0,
              0,
              0,
              1,
            ]
          }
          fillOpacity={1}
          fillRule={1}
          font={Object {}}
          matrix={
            Array [
              1,
              0,
              0,
              1,
              0,
              0,
            ]
          }
          opacity={1}
          originX={0}
          originY={0}
          propList={Array []}
          rotation={0}
          scaleX={1}
          scaleY={1}
          skewX={0}
          skewY={0}
          stroke={null}
          strokeDasharray={null}
          strokeDashoffset={null}
          strokeLinecap={0}
          strokeLinejoin={0}
          strokeMiterlimit={4}
          strokeOpacity={1}
          strokeWidth="1"
          x={0}
          y={0}
        />
        <RNSVGGroup
          fill={
            Array [
              0,
              0,
              0,
              0,
              1,
            ]
          }
          fillOpacity={1}
          fillRule={1}
          font={Object {}}
          matrix={
            Array [
              1,
              0,
              0,
              1,
              0,
              0,
            ]
          }
          opacity={1}
          originX={0}
          originY={0}
          propList={Array []}
          rotation={0}
          scaleX={1}
          scaleY={1}
          skewX={0}
          skewY={0}
          stroke={null}
          strokeDasharray={null}
          strokeDashoffset={null}
          strokeLinecap={0}
          strokeLinejoin={0}
          strokeMiterlimit={4}
          strokeOpacity={1}
          strokeWidth="1"
          x={0}
          y={0}
        />
      </RNSVGSvgView>
    </View>
  </View>
</View>
`;
